nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/java/org/apache/nifi/registry/web/filter/LogoutFilter.java

```diff
@@ -33,19 +34,19 @@ public class LogoutFilter implements Filter {
     private ServletContext servletContext;
 
     @Override
-    public void init(FilterConfig filterConfig) throws ServletException {
+    public void init(FilterConfig filterConfig) {
         servletContext = filterConfig.getServletContext();
     }
 
     @Override
-    public void doFilter(ServletRequest request, ServletResponse response, FilterChain filterChain) throws IOException, ServletException {
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain filterChain) throws IOException {
         final boolean supportsOidc = Boolean.parseBoolean(servletContext.getInitParameter("oidc-supported"));
+
+        final HttpServletResponse httpServletResponse = (HttpServletResponse) response;
         if (supportsOidc) {
-            final ServletContext apiContext = servletContext.getContext("/nifi-registry-api");
-            apiContext.getRequestDispatcher("/access/oidc/logout").forward(request, response);
+            httpServletResponse.sendRedirect("/nifi-registry-api/access/oidc/logout");
```

Comment:
Maybe its a good idea, to move these paths into constant definitions, similar to the other `LogoutFilter` implementations, see "KNOX_LOGOUT_URL"?

Reply:
Thanks, that's worth adjusting.